api/consumers.py

```diff
@@ -210,6 +210,26 @@ async def user_response(self, event):
             text_data=json.dumps({"type": "user_response", "response": event["response"]})
         )
 
+    @database_sync_to_async
+    def fetch_question_results(self, question_id):
+        responses = UserResponse.objects.all().filter(
+            quiz_session__code=self.code, question_id=question_id
+        )
+        answers = responses.distinct("selected_answer")
+        answer_counts = {}
+
+        for response in answers:
+            answer_counts[response.selected_answer] = responses.filter(
+                selected_answer=response.selected_answer
+            ).count()
```

Comment:
You can also replace this with an aggregation in Django's ORM; I'll create a GitHub issue from this comment.